package.json

```diff
@@ -13,7 +13,8 @@
     "dist-all": "build -mwl --x64 --ia32",
     "pack": "build --dir",
     "dist": "build --x64 --ia32",
-    "test": "mocha test"
+    "test": "eslint . && mocha test",
```

Comment:
Hm, it doesn't work for me with `eslint .`, only with `./node_modules/.bin/eslint .`